source/beamline/BioXAS/BioXASBeamline.h

```diff
@@ -196,10 +196,15 @@ class BioXASBeamline : public CLSBeamline
 	/// Returns the detector for the given control, if one has been created and added to the control/detector map.
 	AMBasicControlDetectorEmulator* detectorForControl(AMControl *control) const;
 
-	/// Returns the set of default detectors used in XAS scans.
-	AMDetectorSet* defaultScanDetectors() const { return defaultScanDetectors_; }
-	/// Returns the set of detectors to use as options in scans.
-	AMDetectorSet* scanDetectorsOptions() const { return scanDetectorsOptions_; }
+	/// Returns the set of default detectors added to XAS scans, a subset of defaultXASScanDetectorOptions.
```

Comment:
This subset status of default detectors vs. detector options in managed manually in specific functions for each detector, right? (eg. addGe32Detector() etc.)